app/views/project/build.properties

```diff
@@ -1 +0,0 @@
-sbt.version=1.4.4
```

Comment:
Do we have SBT version defined somewhere else? I wonder why would you delete this file?

Reply:
Mind that this file was under `views`. Probable was added here by error.